lib/beaker/subcommand.rb

```diff
@@ -53,8 +53,11 @@ def initialize(*args)
     class_option :'exclude-tags', :type => :string, :group => 'Beaker run'
     class_option :'xml-time-order', :type => :boolean, :group => 'Beaker run'
 
+    #global class options that should be available for all beaker subcommands
```

Comment:
@ferglor here is the global option for help that will be available for alal subcommands.